secret/1_connecting_to_node/connect.js

```diff
@@ -1,25 +1,27 @@
-const { CosmWasmClient } = require("secretjs");
+const { CosmWasmClient } = require('secretjs');
 
 require('dotenv').config();
 
 const main = async () => {
   // Create connection to DataHub Secret Network node
-  const client = new CosmWasmClient(process.env.SECRET_REST_URL)
+  const client = new CosmWasmClient(process.env.SECRET_REST_URL);
 
   // Query chain ID
   const chainId = await client.getChainId()
+    .catch((err) => { console.error('Could not get chain id:\n', err); });
```

Comment:
```suggestion
    .catch((err) => { throw new Error('Could not get chain id: ' + err); });
```

we're logging the error already in the main catch() at the bottom. I think we only really need to throw the error (then it also doesn't try to execute the rest of the code in main)